drivers/virtio/net/virtio_net.c

```diff
@@ -596,8 +596,12 @@ static int virtio_netdev_rxq_dequeue(struct virtio_net_device *vndev,
 		*netbuf = NULL;
 		return rxq->nb_desc;
 	}
-	if (unlikely((len < (__u32)virtio_net_hdr_size(vndev) + UK_ETH_HDR_UNTAGGED_LEN) ||
-		     len > VIRTIO_PKT_BUFFER_LEN(vndev))) {
+	if (unlikely(
+		(len < (__u32)virtio_net_hdr_size(vndev) +
```

Comment:
Why not keep this on the same line as the `unlikely`?

Reply:
Because it feels much more easier to parse on a quick look.... But yeah I know coding conventions are more important 🙄